ports/mimxrt/mpconfigport.h

```diff
@@ -129,6 +129,12 @@ uint32_t trng_random_u32(void);
 #define MICROPY_PY_MACHINE_SPI              (1)
 #define MICROPY_PY_FRAMEBUF                 (1)
 
+// fatfs configuration used in ffconf.h
+#define MICROPY_FATFS_ENABLE_LFN       (1)
```

Comment:
please indent/align the value it's set to to the same column as the config options above

Reply:
fixed.